apiary.apib

```diff
@@ -2425,6 +2425,20 @@ Register a payment for an invoice.
 
 + Response 204
 
+### invoices.send [POST /invoices.send]
+
+Send an invoice.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `7abb325c-e063-42a4-8fb4-1b730759645a` (string, required)
+        + to: `mary@teamleader.eu`, `john@teamleader.eu` (array[string], required)
+        + subject: `compliments` (string, required)
+        + content: `amazing compliments` (string, required)
```

Comment:
Q: would it also make sense to accept the template to be used for the email?

Reply:
that is for later